memjs/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../tslint.json",
+  "rules": {
+      "no-single-declare-module": false
```

Comment:
Why was it necessary to disable this rule? Could you just remove the ```declare module``` and reduce the first 16 lines to:
```ts
export let Client: { create(serversStr: string, options?: ClientOptions): Client; };
```